CustomKnight/Skin/Base/SkinManager.cs

```diff
@@ -111,9 +111,71 @@ public static class SkinManager{
+            {"Vessel_0",new Vessel("Inventory/Vessel_0",0) },
+            {"Vessel_1",new Vessel("Inventory/Vessel_1",1) },
+            {"Vessel_2",new Vessel("Inventory/Vessel_2",2) },
+            {"Vessel_3",new Vessel("Inventory/Vessel_3",3) },
+            {"Fireball_1",new InvSpell("Inventory/Fireball_1",1) },
+            {"Fireball_2",new InvSpell("Inventory/Fireball_2",2) },
+            {"Quake_1",new InvSpell("Inventory/Quake_1",1) },
+            {"Quake_2",new InvSpell("Inventory/Quake_2",2) },
+            {"Scream_1",new InvSpell("Inventory/Scream_1",1) },
+            {"Scream_2",new InvSpell("Inventory/Scream_2",2) },
+            {"Focus",new InvSpell("Inventory/Focus",0) },
+            {NailArtBG.Name,new NailArtBG() },
+            {"DSlash",new InvNormal("Inventory/DSlash") },
+            {"GSlash",new InvNormal("Inventory/GSlash") },
+            {"CSlash",new InvNormal("Inventory/CSlash") },
+            {"DreamGate",new InvNormal("Inventory/DreamGate") },
+            {"DreamNail_0",new DreamNail("Inventory/DreamNail_0") },
+            {"DreamNail_1",new DreamNail("Inventory/DreamNail_1") },
+            {"InvGeo",new InvNormal("Inventory/Geo") },
+            {"GodFinder_0",new GodFinder("Inventory/GodFinder_0") },
+            {"GodFinder_1",new GodFinder("Inventory/GodFinder_1") },
+            {"GodFinder_2",new GodFinder("Inventory/GodFinder_2") },
+            {"Cloak_1",new InvCloak(1) },
+            {"Cloak_2",new InvCloak(2) },
+            {InvClaw.Name,new InvClaw() },
+            {InvSD.Name ,new InvSD() },
+            {Lantern.Name,new Lantern() },
+            {InvDJ.Name,new InvDJ() },
+            {Tear.Name,new Tear() },
+            {SlyKey.Name,new SlyKey() },
+            {ElegentKey.Name,new ElegentKey() },
+            {WJournal.Name,new WJournal() },
+            {HSeal.Name,new HSeal() },
+            {Idol.Name,new Idol() },
+            {BEgg.Name,new BEgg() },
+            {"Flower_0",new Flower(0) },
+            {"Flower_1",new Flower(1) },
+            {TramPass.Name,new TramPass() },
+            {Ore.Name,new Ore() },
+            {CityKey.Name,new CityKey() },
+            {LoveKey.Name,new LoveKey() },
+            {Brand.Name,new Brand() },
+            {REgg.Name,new REgg() },
+            {SimpleKey.Name,new SimpleKey() },
+            {"Map",new MAQ("Inventory/Map") },
+            {"Quill",new MAQ("Inventory/Quill") },
+            {"MaQ",new MAQ("Inventory/MaQ") },
+            {RelicBG.Name,new RelicBG() },
+            {SpellBG.Name,new SpellBG() },
            // {"PinsScarab", new Pins()}
         };
+        public static GameObject inv = GameCameras.instance.hudCamera.gameObject.FindGameObjectInChildren("Inventory").FindGameObjectInChildren("Inv");
+        public static GameObject equipment = inv.FindGameObjectInChildren("Equipment");
+        public static GameObject invitem = inv.FindGameObjectInChildren("Inv_Items");
```

Comment:
should probably replace this with properties with a backing field that can cache it.